src/SAD/Parser/Token.hs

```diff
@@ -166,22 +166,27 @@ expandTexCmd "lambda" pos whiteSpaceBefore = [makeToken "\\" pos whiteSpaceBefor
 -- list of tex commands here so that they don't use `\`. Note that the fact that this is designed this way makes
 -- it conceptually impossible for the user to configure which tex commands are treated as words on the fly.
 
--- Tex words
-expandTexCmd s pos whiteSpaceBefore | elem (Text.toLower s) texWords = [makeToken ("tex" <> s) pos whiteSpaceBefore]
--- If this is not a predefined command to be expanded, just leave the backslash so that it gets treated as a symbol.
+-- Greek letters
+expandTexCmd s pos whiteSpaceBefore | s `elem` greek = [makeToken ("tex" <> s) pos whiteSpaceBefore]
+-- If this is not a greek letter to be expanded, just leave the backslash so that it gets treated as a symbol.
```

Comment:
Maybe this is a bit nitpicky, but `If this is not a greek letter to be expanded` is inaccurate since we are also excluding things like `\wedge`, `\vee`, etc. Still, it wouldn't be the end of the world if this got merged.

Reply:
Oh, then I misunderstood the code. I'll withdraw the naming.